src/client/components/localisation/controller.ts

```diff
@@ -221,8 +218,8 @@ export class LocalisationController {
     // This camera position hack will not work with orientation/head movement.
     // TODO (Annable): Sync camera position/rotation properly using kinematic chain.
     model.camera.position.set(target.rWTt.x, target.rWTt.y, target.rWTt.z + 0.15)
-    const Rwt = new Quaternion(target.Rwt.x, target.Rwt.y, target.Rwt.z, target.Rwt.w)
-    const heading = new Euler().setFromQuaternion(Rwt).z
+    const Rwt = new THREE.Quaternion(target.Rwt.x, target.Rwt.y, target.Rwt.z, target.Rwt.w)
+    const heading = new THREE.Euler().setFromQuaternion(Rwt).z
```

Comment:
Preferring to prefix `THREE` types when intermixed with our native types.

Reply:
There are other places that we could do this too if this is the way we want to go